docker/build_scripts/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Python versions installed in /opt/python. Could add more, if desired.
+
+# Stop at any error
+set -e
```

Comment:
I'd add `-x` to (which causes the shell to print each command before executing it)